ui.config/src/main/content/jcr_root/apps/acs-commons/config.author/org.apache.sling.jcr.repoinit.RepositoryInitializer-acs-commons-author.config

```diff
@@ -74,5 +74,8 @@ set ACL for acs-commons-file-fetch-service
     allow jcr:read,jcr:versionManagement,rep:write,crx:replicate on /content/dam
     allow jcr:read on /
 end
+
+# Copy Publish URL - AEM Sites Page Editor overlay
+create path (nt:unstructured) /apps/wcm(nt:folder)/core(nt:folder)/content(sling:Folder)/editor(cq:Page)/jcr:content/content/items(sling:OrderedFolder)/content(sling:OrderedFolder)/header/items/headerbar/items/pageinfopopover/items/list/items
```

Comment:
Not sure we need this, but doesnt hurt - and hopefully we could move over to using this instead of all the .content.xml in the future.